src/main/java/com/github/dariobalinzo/ElasticSourceConnector.java

```diff
@@ -121,9 +132,7 @@ public void start(Map<String, String> props) {
     }
 
     private List<Map<String, String>> findTaskFromIndexPrefix(int maxTasks) {
-        List<String> currentIndexes = elasticRepository.catIndices(
-                config.getString(ElasticSourceConnectorConfig.INDEX_PREFIX_CONFIG)
-        );
+        List<String> currentIndexes = topicMonitorThread.topics();
```

Comment:
Maybe you can rename this to indexes instead of topics? topics name are then constructed using a prefix